controllers/reconcilers/logging_installation/logging_installation_reconciler.go

```diff
@@ -360,12 +353,44 @@ func (r *Reconciler) createClusterLogForwarderCr(ctx context.Context, cr *v1.Obs
 			return nil
 		}
 
-		newPipeline.InputRefs = append(newPipeline.InputRefs, "kafka-log-resources")
-
 		_, err = controllerutil.CreateOrUpdate(ctx, r.client, clusterLogForwarder, func() error {
-			clusterLogForwarder.Spec.Pipelines = []v14.PipelineSpec{*newPipeline}
+			clusterLogForwarder.Spec.Inputs = []v14.InputSpec{
+				{
+					Name: "kafka-operator-application-logs",
+					Application: &v14.Application{Namespaces: []string{"redhat-kas-fleetshard-operator", "redhat-managed-kafka-operator",
+						"redhat-kas-fleetshard-operator-qe", "redhat-managed-kafka-operator-qe"}},
+					Infrastructure: nil,
+					Audit:          nil,
+				},
+			}
+
+			clusterLogForwarder.Spec.Outputs = []v14.OutputSpec{
+				{
+					Name: "cloudwatch",
+					Type: "cloudwatch",
+					OutputTypeSpec: v14.OutputTypeSpec{
+						Cloudwatch: &v14.Cloudwatch{
+							Region:  "eu-west-1",
```

Comment:
@StevenTobin this looks like something we might want to make configurable in the future. What do you think?